jax/interpreters/xla.py

```diff
@@ -190,6 +190,17 @@ def _check_nans(name, xla_shape, buf):
         msg = "invalid value (nan) encountered in {}"
         raise FloatingPointError(msg.format(name))
 
+def _make_xla_args(c, *xla_shapes):
```

Comment:
I'd be quite tempted only to tuple up the arguments if there are more than, say, 100 of them. There is a non-zero  cost to building a tuple and it is unnecessary when merely calling `lax.add` or similar.

Reply:
Done.